eeg_face13/dataset_description.json

```diff
@@ -1,5 +1,5 @@
 {
-	"Name": "faceFO",
+	"Name": "Face13-FAC",
```

Comment:
hyphens and underscores are reserved in BIDS so we should avoid using here I think.